Library/Formula/postgres-kmeans.rb

```diff
@@ -0,0 +1,32 @@
+class PostgresKmeans < Formula
+  desc " K-means clustering extension for PostgreSQL"
+  homepage "http://www.pgxn.org/dist/kmeans/doc/kmeans.html"
+  url "http://api.pgxn.org/dist/kmeans/1.1.0/kmeans-1.1.0.zip"
+  sha256 "1fafeb16e0485eb8dfbaaff83469ea593e91844832ce5e673b79ac3f5c915225"
+
+  head "https://github.com/umitanuki/kmeans-postgresql.git"
+
+  depends_on "postgresql"
+
+  def install
+    ENV["USE_PGXS"] = "1"
+    system "make", "install"
```

Comment:
Note that [postgres-kmeans](https://github.com/umitanuki/kmeans-postgresql) does not have configure script.
This extension' s `Makefile` is written in similar way as `pgroonga`.

We can use `pgroonga` like style installation: https://github.com/Homebrew/homebrew/blob/a3f926d0b5384f320247f0555ca609d9839641be/Library/Formula/pgroonga.rb#L20-L25

@rburhum Could you try to use above style in install stage?